README.md

```diff
@@ -23,70 +22,123 @@ It provides fully customizeable image, video, and web page wallpapers that can b
 
 ![s1](https://raw.githubusercontent.com/Komorebi-Fork/komorebi/master/screenshots/collage.jpg)
 
+## Installing
 
-## How do I install Komorebi?
+Komorebi was tested on:
 
-### Packaged install (easy)
+- **Ubuntu** _18.04_
+- **Elementary OS** _5.1.4_
+- **Pop! OS** _20.04_
+- **Fedora** _32_
+- **Manjaro** _20.0.3_
+- **Deepin** _20_
+- **Arch Linux**
+- **Gentoo**
 
-1. Download `Komorebi` from the [Komorebi releases page](https://github.com/Komorebi-Fork/komorebi/releases).
-2. Install Komorebi using your favorite package manager (aka. double click on it)
-3. Launch Komorebi!
+### Debian and derivated (Ubuntu, Deepin, Elementary OS, Pop! OS, etc...)
 
-### Manual Installing (advanced)
+```bash
+sudo apt install meson valac libgtk-3-dev libgee-0.8-dev libclutter-gtk-1.0-dev libclutter-1.0-dev libwebkit2gtk-4.0-dev libclutter-gst-3.0-dev
+```
 
-Run the following:
+`.deb` package coming soon; until then, jump to the [compilling section](#compilling).
+
+### Fedora
+
+**Fedora uses Wayland by default, which Komorebi doesn't support yet. You will have to switch to Xorg for the meantime.**
+
+```bash
+sudo dnf install meson vala gcc-c++ gtk3-devel clutter-devel clutter-gtk-devel clutter-gst3-devel webkit2gtk3-devel libgee-devel gstreamer1-libav
 ```
-sudo add-apt-repository ppa:gnome3-team/gnome3 -y
-sudo add-apt-repository ppa:vala-team -y
-sudo add-apt-repository ppa:gnome3-team/gnome3-staging -y
-sudo apt install meson valac libgtk-3-dev libgee-0.8-dev libclutter-gtk-1.0-dev libclutter-1.0-dev libwebkit2gtk-4.0-dev libclutter-gst-3.0-dev
+
+There is no `.rpm` package yet, jump to the [compilling section](#compilling); if you can package Komorebi for Fedora, go ahead and let us know!
```

Comment:
[compiling section](#compiling) - I'll only note this here but it's in a few places.